optimize/backend/src/main/java/io/camunda/optimize/FilterBeansConfig.java

```diff
@@ -31,6 +32,9 @@
  */
 @Configuration
 public class FilterBeansConfig {
+
+  @Autowired private ConfigurationService configurationService;
```

Comment:
Does `ConfigurationService` need to be added here? I don't see where it is used

Reply:
Oh, it's a change I did then I reverted and I forgot to remove this.
Nice catch, thanks!